examples/map/difference_images.py

```diff
@@ -0,0 +1,64 @@
+"""
+================
+Plotting a difference image
+================
+How to compute and plot a difference image.
+The example uses `sunpy.map.MapSequence` to compute a difference image and then plot it.
+This basic method works for base difference or running difference. Just change whether
+you're subtracting the previous image or the first image in a sequence.
+"""
+###########################################################################
+
+#sphinx_gallery_thumbnail_number = 2
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import astropy.units as u
+from sunpy.net import Fido, attrs as a
+import sunpy.map
+
+###########################################################################
+# First we'll download a couple images and store them in a
+# sunpy.map.MapSequence(). This could be from whatever imager.
+# Here, we use SDO/AIA 304 Å.
+
+instrument = a.Instrument('AIA')
+wave = a.Wavelength(30 * u.nm, 31 * u.nm)
+result1 = Fido.search(a.Time('2015-06-18T00:00:00', '2015-06-18T00:00:10'), instrument, wave)
+result2 = Fido.search(a.Time('2015-06-18T01:03:30', '2015-06-18T01:03:35'), instrument, wave)
+downloaded_files = Fido.fetch(result1, result2)
+maps = sunpy.map.Map(downloaded_files, sequence=True)
+
+###########################################################################
+# Now we'll do a standard plot of the second image just to see it.
+
+plt.figure()
+ax = plt.subplot(projection=maps[1])
+maps[1].plot()
+
+###########################################################################
+# And now we can do take the actual difference.
+
+diff = maps[1].data - maps[0].data
+
+###########################################################################
+# But we have to decide what to do with the metadata. For example, what
+# time does this difference image correspond to? The time of the first or
+# second image? The mean time? You'll have to decide what makes most sense
+# for your application. Here we'll just use the metadata from the second
+# image. Then we can store the difference and header back in a Map.
+
+header = maps[1].fits_header
```

Comment:
You would be better off using `maps[1].meta` here rather than `fits_header`.

Reply:
Done. And updated variable name accordingly.